aiohttp/connector.py

```diff
@@ -69,6 +72,39 @@
     from .tracing import Trace
 
 
+def _convert_hosts_to_addr_infos(
+    hosts: List[Dict[str, Any]]
+) -> List[aiohappyeyeballs.AddrInfoType]:
+    """Converts the list of hosts to a list of addr_infos.
+
+    The list of hosts is the result of a DNS lookup. The list of
+    addr_infos is the result of a call to `socket.getaddrinfo()`.
+    """
+    addr_infos: List[aiohappyeyeballs.AddrInfoType] = []
+    for hinfo in hosts:
+        host = hinfo["host"]
+        is_ipv6 = helpers.is_ipv6_address(host)
+        family = socket.AF_INET6 if is_ipv6 else socket.AF_INET
```

Comment:
is_ipv6 is likely overkill here as we don't need to validate it here, only determine the family

Reply:
We don't need validation, only to get the right family

9e36ed1d830d5d6f02cc79cfd4c2a1777f22fba5